docs/app/experiments/dialog.tsx

```diff
@@ -179,6 +179,7 @@ function ReactSpringTransition(props: { open: boolean; children?: React.ReactEle
   }, [api, open, mounted, setMounted]);
 
   return mounted ? (
+    /* @ts-ignore */
```

Comment:
We could use `@ts-expect-error` and add a description on why is needed.

Reply:
I added an explanation, but I can't use @ts-expect-error as there is no error in React 18 and this code must work in both versions.